Vagrantfile

```diff
@@ -28,6 +30,7 @@ Vagrant.configure("2") do |config|
   config.vm.provider :libvirt do |v|
     v.memory = memory
     v.cpus = cpus
+    v.machine_virtual_size = 60
```

Comment:
Can we apply the same disk size configuration to `:virtualbox` provider too?
https://www.vagrantup.com/docs/disks/usage

Reply:
Sure. It is experimental though.